themes/black-purple-2025/layouts/_partials/recent-posts.html

```
@@ -0,0 +1,31 @@
+{{- $posts := first 6 (where (where .Site.RegularPages "Section" "blog") "Type" "blog") -}}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)